mobile/library/cc/stream.cc

```diff
@@ -11,8 +11,19 @@ Stream::Stream(Envoy::InternalEngine* engine, envoy_stream_t handle)
     : engine_(engine), handle_(handle) {}
 
 Stream& Stream::sendHeaders(RequestHeadersSharedPtr headers, bool end_stream) {
-  envoy_headers raw_headers = rawHeaderMapAsEnvoyHeaders(headers->allHeaders());
-  engine_->sendHeaders(handle_, raw_headers, end_stream);
+  engine_->sendHeaders(
```

Comment:
would it make sense to nix the E-M header classes entirely or do you think that should be a follow-up PR?

Reply:
no I think having API work separate is a sensible choice,  I dislike the complexity of the headers function but I assume at the end of the day once we've done API clean up, it'll just be the responsibility of the caller to create the envoy headers with formatter, and we can remove the function?